### ISSUE TITLE
Benchmark with test fails on SNB

### ISSUE THREAD

**Issue by andreas-abel** [RRZE-HPC/OSACA#70]
The following benchmark leads to a KeyError on SNB. It works on other targets.

`echo 'test %r14,0x38(%rsp)' | osaca --arch SNB -`

Output:

```
Traceback (most recent call last):
  File "/usr/local/bin/osaca", line 11, in <module>
    load_entry_point('osaca==0.3.14', 'console_scripts', 'osaca')()
  File "/usr/local/lib/python3.6/dist-packages/osaca/osaca.py", line 387, in main
    run(args, output_file=args.out)
  File "/usr/local/lib/python3.6/dist-packages/osaca/osaca.py", line 339, in run
    inspect(args, output_file=output_file)
  File "/usr/local/lib/python3.6/dist-packages/osaca/osaca.py", line 293, in inspect
    semantics.add_semantics(kernel)
  File "/usr/local/lib/python3.6/dist-packages/osaca/semantics/arch_semantics.py", line 31, in add_semantics
    self.assign_tp_lt(instruction_form)
  File "/usr/local/lib/python3.6/dist-packages/osaca/semantics/arch_semantics.py", line 282, in assign_tp_lt
    if INSTR_FLAGS.HAS_LD in instruction_form['flags']
  File "/usr/local/lib/python3.6/dist-packages/osaca/semantics/hw_model.py", line 184, in get_load_latency
    return self._data['load_latency'][reg_type] if self._data['load_latency'][reg_type] else 0
  File "/home/andreas/.local/lib/python3.6/site-packages/ruamel/yaml/comments.py", line 753, in __getitem__
    return ordereddict.__getitem__(self, key)
KeyError: 'gpr'

```

**Comment by andreas-abel** [comment]
Same for `echo 'cmovb  0x80(%rsp),%r13d' | osaca --arch SNB -`

**Comment by cod3monk** [comment]
fixed in 0c5ac26.
Thanks for reporting!
